torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -916,18 +927,37 @@ BufHandle TensorExprKernel::bindSymbolicShapeInput(
 
 Tensor TensorExprKernel::bindInput(const torch::jit::Value* input) {
   auto const& t = input->type();
+  auto const& outputs = input->owningGraph()->outputs();
```

Comment:
Why are these changes to `kernel.cpp` in this PR? They are not directly related to "trimming JIT graphs", right?

Reply:
That's correct, this fixes the behavior when a graph input is used directly as an output. We don't usually get such graphs, but that becomes common when we try trimming them.

I'll extract it to a separate PR.